devtools/mog/config.go

```diff
@@ -62,6 +62,29 @@ type fieldConfig struct {
 	FuncFrom   string
 	FuncTo     string
 	// TODO: Pointer pointerSettings
+
+	cfg    structConfig // for dynamic
+	cfgSet bool
```

Comment:
I guess this is a flag to say that we found the referenced struct? Where as previously it would generate code that did not compile?

Reply:
It wouldn't be set if it wasn't actually pointing to a struct type (like `ID string` fields in the e2e test)